examples/acados_python/getting_started/minimal_example_closed_loop.py

```diff
@@ -112,11 +115,13 @@
 
 # closed loop
 for i in range(N):
+
     # solve ocp
     acados_ocp_solver.set(0, "lbx", xcurrent)
     acados_ocp_solver.set(0, "ubx", xcurrent)
 
-    status = acados_ocp_solver.solve()
+    status = acados_ocp_solver.solve(phase=0)
```

Comment:
Honestly I don't like passing the phase to the solve method.
`phase` is just one of the many options affecting the solver, and as such it should be set in the options (before creating the solver), and in the solver itself after its creation
`acados_ocp_solver.set("rti_phase", 0)`
If not handled in this way, the argument list for `solve()` would indefinitely increase as we come up with more stuff.

Reply:
the options can be lumped into a `opts` object - that way the behavior of the call to `solve` would not depend on this additional state of the solver, but rather on the arguments explicitly passed by the user. What do you think?

Reply:
I think both points are valid:
Adding inputs to the `solve` call will add complexity in another place.

Options that can be changed after the creation of the solver are now be gathered here:
https://github.com/acados/acados/blob/35288465716ec7abc538909d77e98dd1e0b70374/interfaces/acados_template/acados_template/AcadosOcpSolver.py#L641

Right now, this also makes
> the behavior of the call to solve would not depend on this additional state of the solver


But having only one API to change these options is definitely desirable IMO.